utils/gen-snapshot-tests.js

```diff
@@ -12,19 +12,11 @@
 const marked = require("marked");
 const fs = require("fs");
 const path = require("path");
-const glob = require("glob");
 const babylon = require("babylon");
 
-const styleguideConfig = require("../styleguide.config.js");
+const {getComponentFilesFromSection} =  require("./styleguidist-config-utils.js");
```

Comment:
I created a file to contain some shared code and updated this script to use that shared code.